accord-core/src/main/java/accord/impl/InMemoryCommandStore.java

```diff
@@ -408,63 +464,39 @@ void update(Ranges add)
         }
     }
 
-    static class CFKEntry extends TxnId
+    enum Loader implements CommandLoader<Command>
```

Comment:
CFK is supposed to be immutable, but the old `CFKEntry` used the global backing commands map and would return updated values, making CFK mutable!  Switched away from mutable loader to immutable loader to better match the expected behavior and what C* does...